integration/benchmarks_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+
+	"github.com/FerretDB/FerretDB/integration/setup"
+	"github.com/FerretDB/FerretDB/integration/shareddata"
+)
+
+func BenchmarkPushdowns(b *testing.B) {
+	ctx, coll := setup.Setup(b, shareddata.AllProviders()...)
+
+	res, err := coll.InsertOne(ctx, bson.D{{}})
+	require.NoError(b, err)
+
+	id := res.InsertedID
+
+	// run benchmark with query pushdown
+	b.Run("ObjectID", func(b *testing.B) {
```

Comment:
```suggestion
	b.Run("Pushdown", func(b *testing.B) {
```

Reply:
I would suggest to leave it as it is. In the future we would implement pushdowns for other types/conditions and it maybe could be useful to benchmark how they affect on our performance. WDYT?

Reply:
ok :hugs: